StrictEmit/ILGeneratorExtensions.Pointers.cs

```diff
@@ -27,6 +27,14 @@ namespace StrictEmit
 {
     public static partial class ILGeneratorExtensions
     {
+        /// <summary>
+        /// Stores a value or an object reference at a supplied address.
+        /// </summary>
+        /// <typeparam name="T">The type.</typeparam>
```

Comment:
The type of what? You can copy the text from the other overload to stay in sync.